app/helpers/country_flag_helper.rb

```diff
@@ -1,10 +1,15 @@
 module CountryFlagHelper
   def country_flag_emoji(two_letter_country_code)
+    return unless two_letter_country_code.match? %r{\A[A-Z][A-Z]\z}
```

Comment:
Wondering whether this is unnecessarily defensive?

Reply:
It was added in because I'd inadvertently called it with a geographical area id that wasn't a country code - it can't handle other format ids - (and they wouldn't correspond to flags anyway). 

If the format is not the expected one - we end up rendering 'unknown character' boxes into the page - just rendering nothing seems like a better way to go.

Reply:
Sounds good

Reply:
Are we 100% sure that the geographical area ids all match the two letter country codes or do we need a mapping between the two? I'd be surprised if they didn't

Reply:
After further discussion with Matt I'm going to switch this to using flags images anyway so I'll address this in that PR